submissions/DarlaMoonlight/html-css-popup/index.html

```diff
@@ -0,0 +1,116 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="utf-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
```

Comment:
Metatag Viewport is mandatory for mobile markup.
https://serpstat.com/ru/blog/chto-takoe-metateg-viewport/
Could you please add it?
`<meta name="viewport" content="width=device-width, initial-scale=1.0">`